pkg/manifest/hooks.go

```diff
@@ -27,34 +28,21 @@ import (
 
 	"k8s.io/helm/pkg/chartutil"
 	"k8s.io/helm/pkg/hooks"
-	"k8s.io/helm/pkg/manifest"
 	"k8s.io/helm/pkg/proto/hapi/release"
 	util "k8s.io/helm/pkg/releaseutil"
 )
 
-var events = map[string]release.Hook_Event{
-	hooks.PreInstall:         release.Hook_PRE_INSTALL,
-	hooks.PostInstall:        release.Hook_POST_INSTALL,
-	hooks.PreDelete:          release.Hook_PRE_DELETE,
-	hooks.PostDelete:         release.Hook_POST_DELETE,
-	hooks.PreUpgrade:         release.Hook_PRE_UPGRADE,
-	hooks.PostUpgrade:        release.Hook_POST_UPGRADE,
-	hooks.PreRollback:        release.Hook_PRE_ROLLBACK,
-	hooks.PostRollback:       release.Hook_POST_ROLLBACK,
-	hooks.ReleaseTestSuccess: release.Hook_RELEASE_TEST_SUCCESS,
-	hooks.ReleaseTestFailure: release.Hook_RELEASE_TEST_FAILURE,
-	hooks.CRDInstall:         release.Hook_CRD_INSTALL,
-}
-
-// deletePolices represents a mapping between the key in the annotation for label deleting policy and its real meaning
-var deletePolices = map[string]release.Hook_DeletePolicy{
-	hooks.HookSucceeded:      release.Hook_SUCCEEDED,
-	hooks.HookFailed:         release.Hook_FAILED,
-	hooks.BeforeHookCreation: release.Hook_BEFORE_HOOK_CREATION,
-}
+// NOTESFILE_SUFFIX that we want to treat special. It goes through the templating engine
+// but it's not a yaml file (resource) hence can't have hooks, etc. And the user actually
+// wants to see this file after rendering in the status command. However, it must be a suffix
+// since there can be filepath in front of it.
+const notesFileSuffix = "NOTES.txt"
 
-// Manifest represents a manifest file, which has a name and some content.
-type Manifest = manifest.Manifest
```

Comment:
Why was this removed?

Reply:
It shouldn't have been, sorry, I rebased and made sure I left the breadcrumbs behind for things that were exported that I moved out of the tiller module.